shell/browser/api/electron_api_cookies.cc

```diff
@@ -171,25 +171,95 @@ base::Time ParseTimeProperty(const std::optional<double>& value) {
+           "Caller did not allow access to the cookie."},
+          {net::CookieInclusionStatus::EXCLUDE_FAILURE_TO_STORE,
+           "The cookie was malformed and could not be stored"},
+          {net::CookieInclusionStatus::EXCLUDE_NONCOOKIEABLE_SCHEME,
+           "Attempted to set a cookie from a scheme that does not support "
+           "cookies."},
+          {net::CookieInclusionStatus::EXCLUDE_OVERWRITE_SECURE,
+           "The cookie would have overwritten a Secure cookie, and was not "
+           "allowed to do so."},
+          {net::CookieInclusionStatus::EXCLUDE_OVERWRITE_HTTP_ONLY,
+           "The cookie would have overwritten an HttpOnly cookie, and was not "
+           "allowed to do so."},
+          {net::CookieInclusionStatus::EXCLUDE_INVALID_DOMAIN,
+           "The cookie was set with an invalid Domain attribute."},
+          {net::CookieInclusionStatus::EXCLUDE_INVALID_PREFIX,
+           "The cookie was set with an invalid __Host- or __Secure- prefix."},
+          {net::CookieInclusionStatus::EXCLUDE_INVALID_PARTITIONED,
+           "Cookie was set with an invalid Partitioned attribute, which is "
+           "only valid if the cookie has a __Host- prefix."},
+          {net::CookieInclusionStatus::EXCLUDE_NAME_VALUE_PAIR_EXCEEDS_MAX_SIZE,
+           "The cookie exceeded the name/value pair size limit."},
+          {net::CookieInclusionStatus::EXCLUDE_ATTRIBUTE_VALUE_EXCEEDS_MAX_SIZE,
+           "Cookie exceeded the attribute size limit."},
+          {net::CookieInclusionStatus::EXCLUDE_DOMAIN_NON_ASCII,
+           "The cookie was set with a Domain attribute containing non ASCII "
+           "characters."},
+          {net::CookieInclusionStatus::
+               EXCLUDE_THIRD_PARTY_BLOCKED_WITHIN_FIRST_PARTY_SET,
+           "The cookie is blocked by third-party cookie blocking but the two "
+           "sites are in the same First-Party Set"},
+          {net::CookieInclusionStatus::EXCLUDE_PORT_MISMATCH,
+           "The cookie's source_port did not match the port of the request."},
+          {net::CookieInclusionStatus::EXCLUDE_SCHEME_MISMATCH,
+           "The cookie's source_scheme did not match the scheme of the "
+           "request."},
+          {net::CookieInclusionStatus::EXCLUDE_SHADOWING_DOMAIN,
+           "The cookie is a domain cookie and has the same name as an origin "
+           "cookie on this origin."},
+          {net::CookieInclusionStatus::EXCLUDE_DISALLOWED_CHARACTER,
+           "The cookie contains ASCII control characters"},
+          {net::CookieInclusionStatus::EXCLUDE_THIRD_PARTY_PHASEOUT,
+           "The cookie is blocked for third-party cookie phaseout."},
+          {net::CookieInclusionStatus::EXCLUDE_NO_COOKIE_CONTENT,
+           "The cookie contains no content or only whitespace."}};
+
+  static_assert(
+      net::CookieInclusionStatus::ExclusionReason::NUM_EXCLUSION_REASONS <= 32,
+      "ExclusionReasons list is out of date and needs to be updated.");
```

Comment:
This check is useful for upstream sizeof tests, but doesn't really protect us here. See the `std::array<>` suggestion above